linters/phpstan/phpstan.test.ts

```diff
@@ -0,0 +1,4 @@
+import { linterCheckTest } from "tests";
+import { skipOS } from "tests/utils";
+
+linterCheckTest({ linterName: "phpstan", skipTestIf: skipOS(["win32", "darwin"]) });
```

Comment:
Why are we skipping darwin for this one? If it's not supported, can you add `supported_platforms` to the linter/tool definition?